platform/darwin/src/MGLPointCollection.h

```diff
@@ -0,0 +1,25 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+#import "MGLShape.h"
+
+@interface MGLPointCollection : MGLShape
+
+/**
+ Creates and returns a `MGLPointCollection` object from the specified set of
+ coordinates.
+ 
+ @param coords The array of coordinates defining the shape. The data in this
+ array is copied to the new object.
+ @param count The number of items in the `coords` array.
+ @return A new multipoint object.
+ */
++ (instancetype)pointCollectionWithCoordinates:(CLLocationCoordinate2D *)coords count:(NSUInteger)count;
+
```

Comment:
We don't have public initializers for any of the other shapes. Why add one in this case? Especially since `MGLPointCollection` currently can't be added to the map as an since there is no representation for that sort of annotation in mbgl core.